Frontend/src/components/Card.tsx

```diff
@@ -36,7 +37,7 @@ export default function Card({ data }: Props) {
         }}
       ></div>
 
-      {data.remaining > 0 ? (
+      {(data as any).maxSeat > 0 ? (
```

Comment:
this should be realtime available seat data (เต็มหรือไม่เต็ม ณ​ ขณะเวลาปัจจุบันไปเลย)

Reply:
maxSeat ตอนนี้มันไม่มีทาง = 0 อยู่ละ เพราะมันไม่ลดเหมือนเมื่อก่อนละ